fmp_crawler/insider_trading_crawler.py

```diff
@@ -71,13 +71,11 @@ async def crawl(self, symbols=None):
             symbols = self.get_symbols_to_crawl()
 
         for symbol in tqdm(symbols, desc="Crawling symbols"):
-            page = 0
-            while True:
-                if not await self.crawl_insider_trading(symbol, page):
+            for page in range(INSIDER_TRADING_MIN_PAGE, INSIDER_TRADING_MAX_PAGE + 1):
```

Comment:
Why INSIDER_TRADING_MAX_PAGE? I didn't see the API doc has this limit. The original while loop is correct, right? Also those constants are the same in another file. Why duplicate the definition?

Reply:
I see. I would recommend using the original `while True` still. Because that method has no downside, right? But when they changed the API's implementation, the logic will break here. After all, the 100 threshold is not even in the document.

Reply:
No intention to block the work. So feel free to proceed. But I don't see why 'while True' isn't the more robust option. As you can always break when it return empty or some response code not 200.

```python
while True:
    try:
        xxxx
    except:
        break

Reply:
Sorry I am not sure if I will make the right changes. I will just merge as-is. Feel free to make a patch to your liking.